v-next/hardhat-keystore/src/internal/keystores/unencrypted-keystore.ts

```diff
@@ -0,0 +1,50 @@
+import type { Keystore, UnencryptedKeystoreFile } from "../types.js";
+
+import { assertHardhatInvariant } from "@ignored/hardhat-vnext-errors";
+
+export class UnencryptedKeystore implements Keystore {
+  readonly #keystoreData: UnencryptedKeystoreFile;
+
+  constructor(keystoreData: UnencryptedKeystoreFile) {
+    this.#keystoreData = keystoreData;
+  }
+
+  public static createEmptyUnencryptedKeystoreFile(): UnencryptedKeystoreFile {
```

Comment:
This is the factory for keystore files, it is used across the tests as well as in the production code.